README.md

```diff
@@ -1,2 +1,109 @@
 # openshift-cli-installer
 Basic openshift install cli wrapper
+
```

Comment:
Please add limitation section that state that we only support IPI install on AWS for now